SS2-Project/Assets/Starstorm2/ScriptableObjects/SS2AssetCollections.cs

```diff
@@ -0,0 +1,99 @@
+﻿using MSU;
+using R2API.ScriptableObjects;
+using RoR2;
+using RoR2.Skills;
+using RoR2.ExpansionManagement;
+using RoR2.EntitlementManagement;
+using System;
+using UnityEngine;
+using UnityEditor;
+namespace SS2
+{
+    //todo: a lot
+    [CreateAssetMenu(fileName = "BodyAssetCollection", menuName = "Starstorm2/AssetCollections/BodyAssetCollection")]
+    public class BodyAssetCollection : AssetCollection
+    {
+        public GameObject bodyPrefab;
+        public GameObject masterPrefab;
+
+        public void OnValidate()
+        {
+            //todo: ensure field values are also contained in AssetCollection.assets
+        }
+    }
+    [CreateAssetMenu(fileName = "SurvivorAssetCollection", menuName = "Starstorm2/AssetCollections/SurvivorAssetCollection")]
+    public class SurvivorAssetCollection : BodyAssetCollection
+    {
+        public SurvivorDef survivorDef;
+    }
+    [CreateAssetMenu(fileName = "MonsterAssetCollection", menuName = "Starstorm2/AssetCollections/MonsterAssetCollection")]
+    public class MonsterAssetCollection : BodyAssetCollection
```

Comment:
Could add the fields to create the Monster's "DirectorCardHolder" for the MixMonsterArtifact